setup.py

```diff
@@ -1,17 +1,17 @@
-from setuptools import setup, find_packages
 import os
 import re
-import subprocess
+import subprocess  # noqa: S404
 import sys
 
+from setuptools import find_packages, setup
 from setuptools.command.test import test as TestCommand
 
 
 class PyTest(TestCommand):
```

Comment:
FWIW this should be removed in favor of calling pytest. Calling `setup.py` directly is deprecated. https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html

Reply:
I didn't look at the contents of the file. I think we're not using setup.py anymore, so presumably it can just be removed.

Reply:
@Dreamsorcerer no, it is still in use, but you can migrate the metadata in it to `setup.cfg` + `pyproject.toml`.